mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule Adminable.MixProject do
   def project do
     [
       app: :adminable,
-      version: "0.2.0",
+      version: "0.3.0",
```

Comment:
We might be able to remove the `harmonium` dependency since you aren't using those helpers anymore.

Reply:
The field module still uses them. I was going to remove them there too but it got complicated. But it will happen eventually